cmd/deploy/deploy.go

```diff
@@ -587,3 +605,18 @@ func (dc *DeployCommand) recreateFailedPods(ctx context.Context, name string) er
 	}
 	return nil
 }
+
+// getVariablesFromCfgmap given a cfgmap this returns the variables as []EnvVar in it
+func decodeConfigMapVariables(stringVariables string) []types.DeployVariable {
```

Comment:
I don't see a test for this. Should we added?

Reply:
i've moved the func to types as is common to the one at destroy...